src/Catalog/CatalogWriterBase.cs

```diff
@@ -148,14 +154,16 @@ protected virtual ResourceSaveOperation CreateSaveOperationForItem(IStorage stor
             // Standard method of operation: if content == null, don't do a thing. Else, write content.
 
             var content = item.CreateContent(Context); // note: always do this first
-            var resourceUri = item.GetItemAddress();
+            var catalogItemUri = item.GetItemAddress();
 
             var operation = new ResourceSaveOperation();
-            operation.ResourceUri = resourceUri;
+            operation.ResourceUri = catalogItemUri;
 
             if (content != null)
             {
-                operation.SaveTask = storage.Save(resourceUri, content, cancellationToken);
+                //the save operation will not be done through the customDomain but directly through the storage
```

Comment:
also space between the text of the comment and the `//`

for example: `// The save...`